api/server/router/grpc/grpc.go

```diff
@@ -39,3 +65,14 @@ func (gr *grpcRouter) initRoutes() {
 		router.NewPostRoute("/grpc", gr.serveGRPC),
 	}
 }
+
+func unaryInterceptor(tp trace.TracerProvider) grpc.UnaryServerInterceptor {
+	withTrace := otelgrpc.UnaryServerInterceptor(otelgrpc.WithTracerProvider(tp), otelgrpc.WithPropagators(propagators))
+
+	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (resp interface{}, err error) {
+		if strings.HasSuffix(info.FullMethod, "opentelemetry.proto.collector.trace.v1.TraceService/Export") {
+			return handler(ctx, req)
+		}
```

Comment:
Why does this particular gRPC request method need to be special-cased to be excluded from tracing? I'd like to see a code comment with the explanation.

Any reason why the special-casing is done with bespoke code rather than [`otelgrpc.WithInterceptorFilter`](https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc#WithInterceptorFilter)?
```go
otelgrpc.WithInterceptorFilter(filters.Not(filters.FullMethodName("/opentelemetry.proto.collector.trace.v1.TraceService/Export")))
```